.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+module.exports = {
```

Comment:
Please outline how any prettier commands (if any) fit into our build. e.g. is `yarn start` enough?